vm_supervisor/vm/firecracker/instance.py

```diff
@@ -142,8 +142,8 @@ async def wait_for_init(self) -> None:
 
         ip = ip.split("/", 1)[0]
 
-        attempts = 10
-        timeout_seconds = 1.0
+        attempts = 30
+        timeout_seconds = 2.0
```

Comment:
```suggestion
        timeout_seconds = 0.5
        timeout_total = 60
        attempts = timeout_total // timeout_seconds
        
```

Reply:
This will increase the ping error logs, andI think that 2 seconds is a good timeout for ping on instances.